test/scale/test-scale.sh

```diff
@@ -312,10 +318,25 @@ generateDeployments() {
     done
 }
 
+generateServices() {
+    local numServices=$1
+    local numDeployments=$2
+    local serviceKind=$3
+
+    for i in $(seq -f "%05g" 1 $numServices); do
```

Comment:
to make `--num-services` optional, I think you'll need to set a default value of 0 somewhere